templates/vsphere/template-variables.rc

```diff
@@ -0,0 +1,22 @@
+# Source: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/docs/getting_started.md
+
+export VSPHERE_USERNAME="vi-admin@vsphere.local"                    # The username used to access the remote vSphere endpoint
+export VSPHERE_PASSWORD="admin!23"                                  # The password used to access the remote vSphere endpoint
```

Comment:
let's avoid this

Reply:
Avoid what?

Reply:
Let's avoid fake credentials. This should be empty or commented out, so that if people forget to add their password, they will fail to generate a cluster template altogether (rather than getting a cluster template with a placeholder password)